modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc

```diff
@@ -3,7 +3,7 @@
 [[composite-databases-delete]]
 = Delete composite databases
 
-You can delete composite databases using either the command `DROP COMPOSITE DATABASE name` or the more general one  `DROP DATABASE name`.
```

Comment:
Not sure this should be 'You'. Probably this paragraph needs some editing.

Reply:
this is the current version (I just removed a space where there were 2 between the words), I'll leave it to the docs team to decide on the `you`